File: fluentcore/src/main/java/com/mcreater/fxfluent/controls/abstractions/Dictable.java
package com.mcreater.fxfluent.controls.abstractions;

import com.mcreater.fxfluent.xaml.ResourceDict;
import com.mcreater.fxfluent.xaml.XamlManager;

public interface Dictable {
    default ResourceDict getResourceDict() {
        return XamlManager.getCurrentDict();
    }
}

File: fluentcore/src/main/java/com/mcreater/fxfluent/controls/abstractions/Dictable.kt
package com.mcreater.fxfluent.controls.abstractions

import com.mcreater.fxfluent.xaml.ResourceDict
import com.mcreater.fxfluent.xaml.XamlManager.Companion.getCurrentDict


interface Dictable {
    val resourceDict: ResourceDict?
        get() = getCurrentDict()
}

